go.mod

```diff
@@ -4,25 +4,28 @@ go 1.13
 
 require (
 	github.com/RoaringBitmap/roaring v0.4.23
-	github.com/blevesearch/bleve_index_api v0.0.0-20201109204916-3d41c3915248
+	github.com/blevesearch/bleve_index_api v0.0.1
 	github.com/blevesearch/blevex v0.0.0-20190916190636-152f0fe5c040
+	github.com/blevesearch/cld2 v0.0.0-20200327141045-8b5f551d37f5 // indirect
 	github.com/blevesearch/go-porterstemmer v1.0.3
-	github.com/blevesearch/scorch_segment_api v0.0.0-20201109215555-85deb08a9c2d
+	github.com/blevesearch/scorch_segment_api v0.0.1
```

Comment:
Wouldn't you have had to set scorch_segment_api's go.mod's bleve_index_api requirement to v0.0.1 before setting it's tag to v0.0.1?

I see it still at ..
```
	github.com/blevesearch/bleve_index_api v0.0.0-20201109204916-3d41c3915248
```

Reply:
damn, seems i made the change but never added it to the commit, get to do it all over again

Reply:
Assuming that you won't need to change anything else here, but delete the tag and recreate it after the commit on scorch_segment_api, this change looks good to me.